docs/content/en/docs/getting-started/optional/autoscaling.md

```diff
@@ -49,18 +49,3 @@ cluster.x-k8s.io/cluster-api-autoscaler-node-group-max-size: <maxCount>
 ```
 
 After deploying the Kubernetes Cluster Autoscaler from upstream or as a [curated package]({{< relref "../../packages/cluster-autoscaler/" >}}).
```

Comment:
Can we bring this up before the code block? It's kind of just hanging here. Also what's the purpose of calling out those annotations? Why do customers need to know that?

Reply:
Also I can't leave a comment on the lines you didn't edit but can we more clearly explain the difference between "Kubernetes Cluster Autoscaler clusterapi cloud provider" and the curated package? Why do we have both?